Java/tests/src/test/java/com/microsoft/recognizers/text/tests/choice/BooleanModelTest.java

```diff
@@ -34,7 +37,11 @@ protected void recognizeAndAssert(TestCase currentCase) {
         // parse
         List<ModelResult> results = recognize(currentCase);
         // assert
-        assertResults(currentCase, results, Arrays.asList(ResolutionKey.Value, ResolutionKey.Score));
+        assertResults(currentCase, results, getKeysToTest(currentCase));
+    }
+
+    private List<String> getKeysToTest(TestCase currentCase) {
+        return Arrays.asList(ResolutionKey.Value, ResolutionKey.Score);
```

Comment:
I think that we can rollback this. It's not necessary to divide this to a separated method.